internal/config/config.go

```diff
@@ -44,6 +45,7 @@ func New() *Config {
 		ExportKeys:    true,
 		Mounts:        make(map[string]string),
 		Notifications: true,
+		Parsing:       true,
```

Comment:
This needs to be added to all legacy configs, too (see `legacy.go`). Otherwise people will surprised when updating.

Reply:
Oh, I would have completely missed it. Done.